platform/ios/src/MGLAnnotationView.h

```diff
@@ -141,6 +141,19 @@ typedef NS_ENUM(NSUInteger, MGLAnnotationViewDragState) {
  */
 @property (nonatomic, assign) BOOL scalesWithViewingDistance;
 
+/**
+ A Boolean value that determines whether the annotation view rotates together
+ with the mapview.
+
+ When the value of this property is `YES` and the map is rotated, the annotation
+ view rotates. This is also the behavior of `MGLAnnotationImage` objects. When the
+ value of this property is `NO` the annotation has its rotation angle fixed.
+
+ The default value of this property is `NO`. Set this property to `YES` if the
+ view’s rotation is important.
+ */
+@property (nonatomic, assign) BOOL rotatesToMatchCamera;
```

Comment:
Thanks for making this change. FYI, I was torn between suggesting `rotatesToMatchCamera` and suggesting a `rotationAlignment` property of type `MGLIconRotationAlignment`, for consistency with MGLSymbolStyleLayer. But I think a Boolean `rotatesToMatchCamera` will end up being more intuitive for developers, since we have no need to support an “auto” value here.